bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -138,6 +184,68 @@
 
 	</thing-type>
 
+	<channel-type id="currentType">
+		<item-type unitHint="A">Number:ElectricCurrent</item-type>
+		<label>Current</label>
+		<description>Actual current in A</description>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="energyType">
+		<item-type unitHint="kWh">Number:Energy</item-type>
+		<label>Energy</label>
+		<description>Today's energy in kWh</description>
+		<state readOnly="true" pattern="%.2f kWh"/>
```

Comment:
Hi, I suspect I cant add my review until I'm assigned it. So adding comments and will see if it lefts me add suggestions :) Thank you for the changes it looks good :) Just a minor point that usually gets picked up is the units in the pattern="...... <unit>" can become "..... %unit%" as the unitHint will default to the correct one, and it then if people change the unit conversion it still presents correctly I believe. Same with the other instances as well in this file.

Reply:
Fixed in commit b9e1dc428b